docs/usage.md

```diff
@@ -140,16 +148,11 @@ Default: `100`
 ## Reference Genomes
 
 ### `--genome`
-The reference genome to use for the analysis, needs to be one of the genome specified in the config file. This is `False` by default and needs to be specified (unless index files are supplied, see below).
-
-_Currently only the human and mouse genomes are fully supported by this pipeline. For other genomes `MACS` and `NGSplot` will not run!_
-
-* Human
-  * `--genome GRCh37`
-* Mouse
-  * `--genome GRCm38`
+The reference genome to use for the analysis, needs to be one of the genome specified in the config file. This is `False` by default and needs to be specified. For example:
```

Comment:
It only needs to be specified if using the genome references specified in the iGenomes config. It's not needed if the user supplies all required references directly on the command line.